script/build-artifact.sh

```diff
@@ -18,7 +18,6 @@ if [ "$OS" = "linux" ]; then
         *muslr*) LIBC="-musl" ;;
         *) LIBC="-unknown" ;;
     esac
-    TAROPTS=("--owner=root" "--group=root")
```

Comment:
Sorry, I think we had a miscommunication. I don't think we want to change the ownership. We don't want different user/group settings to leak into the tarball dependent on the build environment.